packages/shared/src/events/generic-event-emitter.ts

```diff
@@ -94,13 +101,8 @@ interface PlatformEventEmitter {
   removeListener(eventName: string | symbol, listener: (...args: any[]) => void): this;
   off(eventName: string | symbol, listener: (...args: any[]) => void): this;
   removeAllListeners(event?: string | symbol): this;
-  setMaxListeners(n: number): this;
-  getMaxListeners(): number;
   listeners(eventName: string | symbol): Function[];
-  rawListeners(eventName: string | symbol): Function[];
   emit(eventName: string | symbol, ...args: any[]): boolean;
   listenerCount(eventName: string | symbol, listener?: Function): number;
-  prependListener(eventName: string | symbol, listener: (...args: any[]) => void): this;
-  prependOnceListener(eventName: string | symbol, listener: (...args: any[]) => void): this;
   eventNames(): Array<string | symbol>;
```

Comment:
These had to be removed as EventEmitter3 removed these methods.
We do not need these and as `PlatformEventEmitter` is internal, it is not breaking to remove these methods.

https://www.npmjs.com/package/eventemitter3#eventemitter3:

> EventEmitter3 is a high performance EventEmitter. It has been micro-optimized for various of code paths making this, one of, if not the fastest EventEmitter available for Node.js and browsers. The module is API compatible with the EventEmitter that ships by default with Node.js but there are some slight differences:
>
>Domain support has been removed.
We do not throw an error when you emit an error event and nobody is listening.
The newListener and removeListener events have been removed as they are useful only in some uncommon use-cases.
The setMaxListeners, getMaxListeners, prependListener and prependOnceListener methods are not available.
Support for custom context for events so there is no need to use fn.bind.
The removeListener method removes all matching listeners, not only the first.

Reply:
Yep